api/tools/src/main/java/org/pmiops/workbench/tools/ExportWorkspaceData.java

```diff
@@ -97,86 +98,37 @@ WorkspacesApi workspaceApi(Provider<WorkbenchConfig> configProvider) {
     return workspacesApi;
   }
 
+  private WorkspaceDao workspaceDao;
+  private CohortDao cohortDao;
+  private ConceptSetDao conceptSetDao;
+  private DataSetDao dataSetDao;
+  private NotebooksService notebooksService;
+  private WorkspaceFreeTierUsageDao workspaceFreeTierUsageDao;
+
   @Bean
   public CommandLineRunner run(
-      WorkspaceDao workspaceDao,
-      CohortDao cohortDao,
-      ConceptSetDao conceptSetDao,
-      DataSetDao dataSetDao,
-      NotebooksService notebooksService,
-      WorkspaceFreeTierUsageDao workspaceFreeTierUsageDao,
+      WorkspaceDao workspaceDaoArg,
```

Comment:
Is the "Arg" suffix meant to indicate that these are passed-in and not injected? Or can you not do `this.workspaceDao = workspaceDao` because you're not in a constructor?